ts/torch_handler/text_handler.py

```diff
@@ -27,9 +28,25 @@ def __init__(self):
     def initialize(self, ctx):
         super(TextHandler, self).initialize(ctx)
         self.initialized = False
-        self.source_vocab = torch.load(self.manifest['model']['sourceVocab'])
+        source_vocab = self.manifest['model']['sourceVocab'] if 'sourceVocab' in self.manifest['model'] else None
+        if source_vocab:
+            # Backward compatibility
```

Comment:
@dhaniram-kshirsagar Please provide details of actual tests run for backward compatibility

Reply:
The old mar is part of ci and it is working fine with this build.